lib/venue.rb

```diff
@@ -1,2 +1,47 @@
 class Venue
+  attr_accessor :bird_type, :capacity, :capacity
+
+  def initialize(bird_type, capacity)
+    @bird = bird_type
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def name
+    @bird
+  end
+
+  def capacity
+    @capacity
+  end
+
+  def patrons
+    @patrons
+  end
+
+  def add_patron(name)
+    @patrons << name
+  end
+
+  def yell_at_patrons
+    yelled_names = []
+    patrons.each do |patron|
+      yelled_names << patron.upcase
+    end
+    return yelled_names
```

Comment:
You do not need to have the keyword `return` here. You can just write `yelled_names` as the last line